### PR TITLE
V2

### PR DESCRIPTION
**Pull request by rosskukulinski**
Changes necessary to run N|Solid 2.0.0 in Docker Compose

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,29 +1,31 @@
...
+    container_name: nsolid.console
+    environment:
+      - NODE_DEBUG=nsolid
+      - NSOLID_CONSOLE_STORAGE_URL=https://storage:4000
+    links:
+      - storage
+    ports:
+      - 3000:3000
+  # app:
+  #   net: "nsolid"
```
Comment (retrohacker) [inline docker-compose.yml]: `net` shouldn't be needed anymore

Comment (rosskukulinski) [inline docker-compose.yml]: thanks @retrohacker

**Comment by rosskukulinski** [comment]
@nw could you please test my changes?

File: docker-compose.yml [hunk 2]
```diff
@@ -1,30 +1,30 @@
...
+    container_name: nsolid.console
+    environment:
+      - NODE_DEBUG=nsolid
+      - NSOLID_CONSOLE_STORAGE_URL=https://storage:4000
+    links:
+      - storage
+    ports:
+      - 3000:3000
+  # app:
+  #   image: nodesource/nsolid:argon
```
Comment (nw) [inline docker-compose.yml]: image should be `nodesource/nsolid:argon-2.0.0`. Maybe consider defaulting to `nodesource/nsolid:boron-2.0.0` ?

**Comment by rosskukulinski** [comment]
@nw PTAL

**Pull request merged**